cmd/ping.go

```diff
@@ -22,7 +25,14 @@ can connect to Kong's Admin API or not.`,
 		}
 		conf, err := client.Root(nil)
 		if err != nil {
-			return errors.Wrap(err, "connecting to kong")
+			config.Workspace = pingWorkspace
+			version, err := kongVersion(config)
+			if err != nil {
+				return errors.Wrap(err, "reading Kong version")
+			}
+			fmt.Println("Successfully connected to Kong!")
+			fmt.Println("Kong version: ", version)
+			return nil
```

Comment:
The call on line 26 is not needed in this case since client.Root() is already called in `kongVersion()`.
Please simplify this.

Reply:
Hello @hbagdi , ping command simplified!